src/IW/App/Error.hs

```diff
@@ -118,15 +123,22 @@ data AppErrorType
     | UrlDownloadFailed Url
     deriving (Show, Eq)
 
--- | Map the @github@ library's @Error@ type into AppErrorType.
+newtype CabalPError = CabalPError { unCabalPError :: PError }
+    deriving (Show)
+
+instance Eq CabalPError where
+    (CabalPError (PError pos1 str1)) == (CabalPError (PError pos2 str2)) =
+        pos1 == pos2 && str1 == str2
```

Comment:
Isn't it the same as what the derived instance would look like?

Reply:
@vrom911 I thought so too, but for some reason the `PError` type from the `Cabal` package isn't an instance of `Eq`. So I thought that it would still be fine I could just implement it. But when I implement an `Eq` instance for `PError` it compiles but gives me the `orphan-instances` warning. I learned that this happens when the instance isn't declared in the original module that the type comes from. So I had to use a newtype wrapper, in this case `CabalPError`, and implement the instance manually. Let me know if there is a better way to do this.

Reply:
Got it, thanks for the clarification!